docs/datacard.py

```diff
@@ -20,6 +23,7 @@
     ),
 ]
 CACHE = audeer.mkdir('./cache')
+BUILD = audeer.path('../build/html')
```

Comment:
Is there a reason not to normalize the paths?

Reply:
I cannot comment on old code. But in line 357 there is an instance where double quotes are used whereas in the rest of the code it is single quotes.

`", ".join`

Reply:
I created https://github.com/audeering/audbcards/issues/2 to track the build folder sphinx argument problem.